ci-scripts/circleci_wait.sh

```diff
@@ -3,7 +3,7 @@
 
 CMD="$*"
 
-TIMEOUT=3700
+TIMEOUT=4300
```

Comment:
Was it a one-time hiccup that the timeout wasn't enough or do you think it's a frequent problem?

Reply:
@AronNovak 

It seems that whenever we exceed 1h execution, we get timeout anyways.
So, this change is not effective. Saw tasks fail with this increased timeout.
All state 1h execution time. For example: https://app.circleci.com/pipelines/github/TIP-Global-Health/eheza-app/4742/workflows/022cde6a-0985-44b5-9143-90f92554c68b/jobs/16322

![Selection_875](https://github.com/user-attachments/assets/1c56f7fc-fabf-4060-82a8-2a15638426f0)

Reply:
@anvmn if needed, let's open an issue for me and I can investigate a bit. Thanks for the details!